flight/Modules/Actuator/actuator.c

```diff
@@ -458,13 +347,14 @@ static void actuatorTask(void* parameters)
  *Process mixing for one actuator
  */
 float ProcessMixer(const int index, const float curve1, const float curve2,
-		   const MixerSettingsData* mixerSettings, ActuatorDesiredData* desired, const float period)
+		   ActuatorDesiredData *desired)
 {
-	float result = (((float)MixerEntry_t[index].vector[MIXERSETTINGS_MIXER1VECTOR_THROTTLECURVE1] * curve1) +
-	                ((float)MixerEntry_t[index].vector[MIXERSETTINGS_MIXER1VECTOR_THROTTLECURVE2] * curve2) +
-	                ((float)MixerEntry_t[index].vector[MIXERSETTINGS_MIXER1VECTOR_ROLL] * desired->Roll) +
-	                ((float)MixerEntry_t[index].vector[MIXERSETTINGS_MIXER1VECTOR_PITCH] * desired->Pitch) +
-	                ((float)MixerEntry_t[index].vector[MIXERSETTINGS_MIXER1VECTOR_YAW] * desired->Yaw)) * (1.0f / MULTIROTOR_MIXER_UPPER_BOUND);
+	int8_t *vector = GetMixerVector(index);
```

Comment:
If we really care about performance it wouldn't be insane to completely unroll the loops and waste a little bit of flash which would be faster than the old code.

I'm biased to not care:  On Cortex-M4 performance doesn't matter; on M3 this compares to ~50 cycles for each of the below multiplies (6) and ~20 cycles for each of the adds (4). So maybe it's a 10% penalty for runtime for this one function?